geocoder/mapquest_reverse.py

```diff
@@ -31,9 +31,11 @@ class MapquestReverse(MapquestQuery):
     provider = 'mapquest'
     method = 'reverse'
 
+    _URL = 'http://www.mapquestapi.com/geocoding/v1/reverse'
+
     def _build_params(self, location, provider_key, **kwargs):
         return {
-            'key': self._get_api_key(mapquest_key),
+            'key': provider_key,
```

Comment:
interesting that you need to shortcut the default behaviour here... what was the motivation ?

Reply:
Correct me if I'm wrong, but I believe that was a bug:

Calling it here (in mapquest_reverse.py) was preventing users from calling the method with the key as an argument.
Furthermore the base class (MultipleResultsQuery) already calls \_get_api_key() in its \_\_init__ method, ensuring that either a key was passed as an argument or that it is available through environment variables.

Reply:
yes you are perfectly right 👍